### PR TITLE
Answers core adapter tests

### PR DESCRIPTION
**Pull request by cea2aj**
Test the adapters from the answers core models the the SDK models

J=SLAP-891
Test=auto

Run the tests and confirm they pass

### CODE REVIEW

File: tests/core/models/directanswer.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import DirectAnswer from '../../../src/core/models/directanswer';
+
+it('constructs an direct answer from an answers-core direct answer', () => {
```
Comment (creotutar) [inline tests/core/models/directanswer.js]: nit: `a direct answer`

**Comment by creotutar** [comment]
LGTM from what i know. Is Autocomplete being done in another PR?

File: tests/core/models/navigation.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import Navigation from '../../../src/core/models/navigation';
+
+it('constructs a navigation model from an answers-core vertical results array', () => {
+  const verticalResultsArray = [
+    {
+      verticalKey: 'faqs'
+    }, {
+      verticalKey: 'locations'
```
Comment (oshi97) [inline tests/core/models/navigation.js]: formatting is a little funny, think the open braces should get a new line or the whole object should be a single line

File: tests/core/models/result.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import Result from '../../../src/core/models/result';
+
+it('constructs a result from an answers-core result', () => {
+  const fullDescription = 'Blueberries are perennial flowering plants with blue or purple berries. They are classified in the section Cyanococcus within the genus Vaccinium. Vaccinium also includes cranberries, bilberries, huckleberries and Madeira blueberries. Commercial blueberries—both wild (lowbush) and cultivated (highbush)—are all native to North America.';
+  const truncatedDescriptionWithHighlight = '<strong>Blueberries</strong> are perennial flowering plants with blue or purple berries. They are classified in the section Cyanococcus within the genus Vaccinium. Vaccinium also includes cranberries, bilberries, huckleberries and Madeira...';
```
Comment (oshi97) [inline tests/core/models/result.js:5]: yum!

File: tests/core/models/universalresults.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import coreUniversalResponse from '../../fixtures/core/coreuniversalresponse.json';
+import sdkUniversalResults from '../../fixtures/core/sdkuniversalresults.json';
+import UniversalResults from '../../../src/core/models/universalresults';
+
+it('constructs a universal results model from an answers-core universal results model', () => {
```
Comment (oshi97) [inline tests/core/models/universalresults.js:5]: could we add a comment here and/or in src/.../universalresults that urls is deprecated? We have verticalPages config now, which does the same thing (see here) https://github.com/yext/answers/blob/da4f923fa33a6974392f71106ae8fcf12429b738/src/ui/components/search/searchcomponent.js#L622

**Review by oshi97 — COMMENTED** [review]
mostly lgtm!

**Comment by cea2aj** [comment]
> LGTM from what i know. Is Autocomplete being done in another PR?

yep, autocomplete is coming after it's finished in the core

File: tests/core/models/navigation.js (hunk 2 above)
Comment (cea2aj) [reply tests/core/models/navigation.js]: going with single line

File: tests/core/models/universalresults.js (hunk 4 above)
Comment (cea2aj) [reply tests/core/models/universalresults.js:5]: If it's deprecated, is now a good time to remove it? If not, i'll add a comment in src/.../universalresults.js

Comment (oshi97) [reply tests/core/models/universalresults.js:5]: Yeah I think it would be great to remove it!

Comment (oshi97) [reply tests/core/models/universalresults.js:5]: the "less temporary hack for DEMO"s the better

Comment (oshi97) [reply tests/core/models/universalresults.js:5]: If it's too much we can do it in another PR though

Comment (cea2aj) [reply tests/core/models/universalresults.js:5]: ya, let's do this in another PR

**Comment by oshi97** [comment]
will take a look once conflicts are resolved!

File: src/core/core-adapter.js [hunk 5]
```diff
@@ -335,7 +335,7 @@ export default class CoreAdapter {
         this.globalStorage.delete(StorageKeys.QUERY_TRIGGER);
 
         const exposedParams = this._getOnUniversalSearchParams(
-          data[StorageKeys.UNIVERSAL_RESULTS].sections,
+          data.universalResults.sections,
```
Comment (oshi97) [inline src/core/core-adapter.js]: I'm also (presumably?) not a huge fan of using storage keys when `data` isn't a storage system but we may as well be consistent

Comment (cea2aj) [reply src/core/core-adapter.js]: ah, you're right. I missed this one during the merge

**Review by oshi97 — APPROVED** [review]
lgtm!

**Pull request merged**